include/dcmqi/ConverterBase.h

```diff
@@ -260,6 +264,32 @@ namespace dcmqi {
       return 0;
     }
 
+    // AF: I could not quickly figure out how to template this function over image type - suggestions are welcomed!
```

Comment:
What I really want is to template over `ImageType` as `T`, and then refer to the argument as `T::Pointer`, but this does not compile.